interfaces/Libraries.ts

```diff
@@ -184,6 +184,34 @@ nlp = spacy.load("${nameWithoutNamespace(model.modelId)}")
 import ${nameWithoutNamespace(model.modelId)}
 nlp = ${nameWithoutNamespace(model.modelId)}.load()`;
 
+const speechbrainASR = (model: ModelData) =>
+`from speechbrain.pretrained import EncoderDecoderASR
+
+asr_model = EncoderDecoderASR.from_hparams(source="${model.modelId}")
+asr_model.transcribe_file("file.wav")`;
+
+const speechbrainEnhancement = (model: ModelData) =>
+`from speechbrain.pretrained import SpectralMaskEnhancement
+
+enhance_model = SpectralMaskEnhancement.from_hparams(source="${model.modelId}")
+enhanced = enhance_model.enhance_file("file.wav")`;
+
+const speechbrainSeparator = (model: ModelData) =>
+`from speechbrain.pretrained import SepformerSeparation
+
+separator_model = SepformerSeparation.from_hparams(source="${model.modelId}")
+est_sources = separator_model.separate_file("file.wav")`;
+
+const speechbrain = (model: ModelData) => {
+	if (model.tags.includes("automatic-speech-recognition")){
+		return speechbrainASR(model);
+	} else if (model.tags.includes("speech-enhancement")) {
+		return speechbrainEnhancement(model);
+	} else if (model.tags.includes("source-separation")) {
```

Comment:
```suggestion
	} else if (model.tags.includes("audio-source-separation")) {
```
this is the currently defined tag in [interfaces/Types.ts](https://github.com/huggingface/huggingface_hub/blob/main/interfaces/Types.ts)